src/scantree/_path.py

```diff
@@ -1,25 +1,22 @@
-from __future__ import print_function, division
-
 import os
-from os import scandir
+from os import DirEntry, scandir
```

Comment:
This could fix the windows issue (but i don't have a way to test it )

based on https://docs.python.org/3.12/library/posix.html ... part `Do not import this module directly`